src/System.Security.Cryptography.Pkcs/ref/System.Security.Cryptography.Pkcs.cs

```diff
@@ -85,6 +85,7 @@ public sealed partial class CmsSigner
         public CmsSigner() => throw null;
         public CmsSigner(SubjectIdentifierType signerIdentifierType) => throw null;
         public CmsSigner(System.Security.Cryptography.X509Certificates.X509Certificate2 certificate) => throw null;
+        public CmsSigner(System.Security.Cryptography.AsymmetricSignatureFormatter formatter, System.Security.Cryptography.X509Certificates.X509Certificate2 certificate) => throw null;
```

Comment:
Should formatter be after the certificate? With a ctor containing `X509Certificate2` already, adding `formatter` afterwards might make better sense for overloading order.

Reply:
I'll check if we got any convention for that, I do not have any preference here - I thought AsymmetricSignatureFormatter felt like more important so I put it first